test-js/unit/prompting/PromptPanelSpec.js

```diff
@@ -546,14 +546,20 @@ define([ 'dojo/number', 'dojo/i18n', 'common-ui/prompting/PromptPanel',
             }
           });
           comp.type = "ScrollingPromptPanelLayoutComponent";
-          spyOn(window, "$");
+
+          // TODO: Find alternative for:
+          //spyOn(window, "$");
```

Comment:
Ok. I just need to be sure it is the same jQuery instance that the source code is using. I say this because in some places it is common-ui/jquery-clean and in others cdf/lib/jquery